### PR TITLE
LPS-79362 Use entries instead of models in all views that show the configuration category menu

### PR DESCRIPTION
**Pull request by JorgeFerrer**

### CODE REVIEW

**Comment by JorgeFerrer** [comment]
ci:test

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/EditConfigurationMVCRenderCommand.java [hunk 1]
```diff
@@ -109,6 +112,15 @@ public String render(
 				ConfigurationAdminWebKeys.CONFIGURATION_CATEGORY_MENU_DISPLAY,
 				configurationCategoryMenuDisplay);
 
+			ConfigurationEntry configurationEntry =
+				new ConfigurationModelConfigurationEntry(
+					configurationModel, PortalUtil.getLocale(renderRequest),
```
[the remark this answers is not in the document]
Comment (JorgeFerrer) [reply modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/EditConfigurationMVCRenderCommand.java]: Good catch, thanks!

**Comment by drewbrokke** [comment]
PR sent to https://github.com/brianchandotcom/liferay-portal/pull/56888 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**